ingestion/src/metadata/ingestion/source/database/databricks.py

```diff
@@ -121,7 +133,56 @@ def get_columns(self, connection, table_name, schema=None, **kw):
     return result
 
 
+@reflection.cache
+def get_schema_names(connection, **kw):
+    # Equivalent to SHOW DATABASES
+    _ = connection.execute(f"USE CATALOG {kw.get('database')}")
```

Comment:
I feel here no need to do `_ = `   you can just do `connection.execute`

Reply:
Yes, sure let me do that. way.